vignettes/cgdsMigration.Rmd

```diff
@@ -169,59 +316,148 @@ getDataByGenes(
     genes = c("NF1", "TP53", "ABL1"),
     by = "hugoGeneSymbol",
     molecularProfileIds = "gbm_tcga_pub_mrna"
-)
+) %>%
+    .[[1]] %>%
+    select(hugoGeneSymbol, value, sampleId) %>%
+    tidyr::spread(key = hugoGeneSymbol, value = value) %>%
+     datatable(options = list(pageLength = 5))
+
+```
+
+**(3) `cBioPortalData` function**
+It is important to note that end users who wish to obtain the data as
+easily as possible should use the main `cBioPortalData` function:
+
+NOT WORKING
+```{r, eval=FALSE}
+cBioPortalData(
+    api = cbio,
+    studyId = "gbm_tcga_pub",
+    genes = c("NF1", "TP53", "ABL1"), 
+    by = "hugoGeneSymbol",
+    molecularProfileIds = "gbm_tcga_pub_mrna"
+) %>%
+    .[[1]] %>%
+    datatable(options = list(pageLength = 5))
 ```
 
-# Mutation data
+# Mutation Data {.tabset .tabset-fade .tabset-pills}
 
-## cgds
+## `cgdsr`
 
-```{r,eval=FALSE}
+```{r}
 getMutationData.CGDS(
     x = cgds, 
     caseList = "getMutationData",
     geneticProfile = "gbm_tcga_pub_mutations",
     genes = c("NF1", "TP53", "ABL1")
-)
+) %>%
+    select(entrez_gene_id, gene_symbol, case_id, amino_acid_change, reference_allele, variant_allele, functional_impact_score, everything()) %>%
+    datatable(options = list(pageLength = 5))
 ```
 
-## cBioPortalData
 
-Similar to `molecularData`, mutation data can be obtained with the
-`mutationData` function or the `getDataByGenes` function.
+## `cBioPortalData`
+
+### Three ways to get molecular Data
 
+**`mutationData` funtion**
+
+Similar to `molecularData`, `mutationData` function  can query mutation data.
 ```{r}
 mutationData(
     api = cbio,
     molecularProfileIds = "gbm_tcga_pub_mutations",
-    entrezGeneIds = entrez,
-    sampleIds = unlist(allsamps)
-)
+    #by = "hugoGeneSymbol",
+    #genes = c("NF1", "TP53", "ABL1"),
+    entrezGeneIds = c("7157", "25", "4763"), # our_entrez, 
+    sampleIds = unlist(samplesInSampleLists(api = cbio,
+                                                      sampleListIds = "gbm_tcga_pub_all"))
+) %>%
+    .[[1]] %>%
+    select(entrezGeneId, sampleId, proteinChange,keyword, referenceAllele, variantAllele, variantType, everything()) %>%
+     datatable(options = list(pageLength = 5))
 ```
 
+**(2) `getDataByGene` function**
+
+ Mutation data can also queried by  the `getDataByGenes` function.
+
 ```{r}
 getDataByGenes(
     api = cbio,
     studyId = "gbm_tcga_pub",
     genes = c("NF1", "TP53", "ABL1"),
     by = "hugoGeneSymbol",
     molecularProfileIds = "gbm_tcga_pub_mutations"
+) %>%
+    .[[1]] %>%
+        select(entrezGeneId, sampleId, proteinChange,keyword, referenceAllele, variantAllele, variantType, everything()) %>%
+    datatable(options = list(pageLength = 5))
+```
+
+**(3) `cBioPortalData` function**
+NOT WORKING
```

Comment:
Again, this is working. Please check your version of `cBioPortalData` and make sure it is the latest.